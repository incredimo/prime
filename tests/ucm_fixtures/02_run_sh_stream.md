```ucm run#r1 sh=true timeout="2s"
echo PRIME
```