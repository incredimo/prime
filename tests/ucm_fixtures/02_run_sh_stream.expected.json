{
  "id": "r1",
  "verb": "run",
  "attrs": {
    "sh": true,
    "timeout": "2s"
  },
  "body": "echo PRIME"
}